.github/workflows/Github_pytest.yml

```diff
@@ -60,7 +60,11 @@ jobs:
         working-directory: ./tests
       - name: Serial fortran tests and non-tagged tests
         run: |
-          python -m pytest -rx -m "not parallel and not c" --ignore=printers --ignore=symbolic --ignore=ndarrays
+          python -m pytest -rx -m "not parallel and not c and not python" --ignore=printers --ignore=symbolic --ignore=ndarrays
```

Comment:
I've just checked, one can write `not (c or python)`

Reply:
Done in 4d42e8d